addons/crm/models/crm_lead.py

```diff
@@ -1920,9 +1920,6 @@ def _notify_by_email_prepare_rendering_context(self, message, msg_vals=False, mo
             message, msg_vals, model_description=model_description,
             force_email_company=force_email_company, force_email_lang=force_email_lang
         )
-        if self.date_deadline:
-            render_context['subtitles'].append(
-                _('Deadline: %s', self.date_deadline.strftime(get_lang(self.env).date_format)))
```

Comment:
If we only call the parent function without changing its parameters and/or the output value, we can maybe completely remove that method?

When calling that function on a recordset of `crm.lead`, the system should call the function defined in the mail.thread mixin with the lead context.

Reply:
@julien-banken yeah, don't know how i missed that :)